tokenserver/views.py

```diff
@@ -96,6 +96,13 @@ def valid_assertion(request):
             raise _unauthorized("invalid-timestamp")
         raise _unauthorized("invalid-credentials")
 
+    try:
+        tokenVerified = assertion['idpClaims']['fxa-tokenVerified']
+    except KeyError:
+        tokenVerified = True
```

Comment:
`assertion['idpClaims'].get('fxa-tokenVerified', True)` failed for tests where `idpClaims` isn't set, so I've instead replaced the try/except with a double `get`. Or is there a better way?

Also, the double `get` edged me past the 80th column so I had to introduce a line break. Not sure whether the indentation I went for on my continuation line is okay or visually abhorrent.